docs/filling out config.md

```diff
@@ -2,7 +2,7 @@
 
 ## Host
 
-Host indicates to our program where to host the data from the Excel file, whether that is 
+Host indicates to our program where to host the data from the Excel file, whether that is
```

Comment:
This is unclear to me. I would simplify and say something like "Host indicates the CRIPT instance that you want to upload your data to (e.g., criptapp.org)."